pages/governance/proposing_a_new_market.md

```diff
@@ -0,0 +1,130 @@
+  - Defines the exponent on the reference price as `log_price:=FLOOR(log10(reference_price))`. For example `log_price:=FLOOR(log10(40,000))=4`
+- Exchanges required for oracles (`exchange_config_json`)
+- Liquidity Tier (these are based on the liquidity tiers recommended, so mappings could change if these tiers change)
+  - 0: Large Cap (BTC/ETH)
+  - 1: Mid-Cap
+  - 2: Long-tail
+
+### Outputs
+
+| Message Type | Field | Description | Value |
+|--------------|-------|-------------|-------|
+| `MsgCreateOracleMarket` | `exponent` | TODO: definition | `p-9` |
+| `MsgCreateOracleMarket` | `min_exchanges` | Used for an index price to be valid. | `3` |
+| `MsgCreateOracleMarket` | `min_price_change_ppm` | The minimum amount the index price has to change for an oracle price update to be valid. | Liquidity Tier 0: `1000` <br> Liquidity Tier 1: `2500` <br> Liquidity Tier 2: `4000` |
+| `MsgCreatePerpetual` | `atomic_resolution` | L the exponent for converting an atomic amount (`size = 1`) to a full coin. |  `-6 - p` |
+| `MsgCreatePerpetual`  | `default_funding_ppm` | The default funding payment if there is no price premium. In parts-per-million. | `0` |
+| `Msg[Update/Create]ClobPair` | `quantum_conversion_exponent` | `10^quantum_conversion_exponent` gives the number of quote quantum traded per base quantum. | `-9` |
+| `Msg[Update/Create]ClobPair` | `subticks_per_tick` | Defines the tick size of the orderbook by defining how many subticks are in one tick. | Liquidity Tier 0: `100000` <br> Liquidity Tier 1 and 2 : `1000000` |
+| `Msg[Update/Create]ClobPair` | `step_base_quantums` | (aka step size): min increment in the size of orders (number of coins) on the CLOB in base quantums. | `1000000` |
+
+## Example Proposal Json
+
+Below is an example proposal JSON file to propose adding `BTC-USD` as a new perpetual market (if it had not been added yet).
+
+```json
+{
+    "title": "Add BTC-USD perpetual market",
+    "deposit": "10000000000000000000000adv4tnt",
+    "summary": "Add the `x/prices`, `x/perpetuals` and `x/clob` parameters needed for a BTC-UTC perpetual market. Create the market in `INITIALIZING` status and transition it to `ACTIVE` status after 3600 blocks."
+    "messages": [
+      {
+        "@type": "/dydxprotocol.prices.MsgCreateOracleMarket",
+        "authority": "dydx10d07y265gmmuvt4z0w9aw880jnsr700jnmapky",
+        "params": {
+            "exchange_config_json": '{"exchanges":[{"exchangeName":"Binance","ticker":"BTCUSDT","adjustByMarket":"USDT-USD"},{"exchangeName":"Bybit","ticker":"BTCUSDT","adjustByMarket":"USDT-USD"},{"exchangeName":"CoinbasePro","ticker":"BTC-USD"},{"exchangeName":"Huobi","ticker":"btcusdt","adjustByMarket":"USDT-USD"},{"exchangeName":"Kraken","ticker":"XXBTZUSD"},{"exchangeName":"Kucoin","ticker":"BTC-USDT","adjustByMarket":"USDT-USD"},{"exchangeName":"Mexc","ticker":"BTC_USDT","adjustByMarket":"USDT-USD"},{"exchangeName":"Okx","ticker":"BTC-USDT","adjustByMarket":"USDT-USD"}]}',
```

Comment:
Is something wrong with the JSON? Why are these lines highlighted? 
<img width="997" alt="Screenshot 2023-12-13 at 3 51 09 PM" src="https://github.com/dydxprotocol/v4-documentation/assets/117109716/6f344174-2752-46a9-9a17-b12f77a6b1fc">

Reply:
Fixed